api-example.html

```diff
@@ -2,7 +2,8 @@
 <html>
     <head>
         <link href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
-        <link href="/c4x/edX/JS101/asset/vectordraw.css" rel="stylesheet" type="text/css" />   <!-- Default styles can be overriden here -->
+        <link href="/c4x/edX/JS101/asset/vectordraw.css" rel="stylesheet" type="text/css" />
```

Comment:
@mtyaka README talsk about `DavidsonNext/Ann101/`, and this file uses `edX/JS101` - might make sense to mention that too (or change to DavidsonNext as well)